.eslintrc

```
@@ -0,0 +1,6 @@
+{
+  "extends": ["@react-native-community"],
+  "rules": {
+    "no-console": 2
```

Can we use the string variant instead of a number? Easier to read:
```suggestion
    "no-console": "error"
```